packages/blockchain-info-components/src/Images/Images.js

```diff
@@ -14,6 +14,7 @@ import buySellSellBtc from './img/buy-sell-sell-btc.svg'
 import buySellSellEth from './img/buy-sell-sell-eth.svg'
 import chartPlaceholder from './img/chart-placeholder.png'
 import checkHelper from './img/check-helper.svg'
+import coinifyLanding from './img/coinify-landing.svg'
```

Comment:
svg degrade perf on slower networks because of being too large, we should consider converting everything to compressed `png` sometime.